Tasks/NugetPublisher/NuGetPublisher.ps1

```diff
@@ -81,26 +81,29 @@ if($connectedServiceName -and $useExternalFeed)
 }
 elseif($feedName -and (-not $useExternalFeed))
 {
+
     Write-Verbose "Using provided feed URL"
     $nugetServer = $feedName
-
-    #check if nuget config exists
-    if(-not (Test-Path -Path $tempNuGetConfigPath))
+    
+    if (-not [URI]::IsWellFormedUriString($nugetServer, [UriKind]::Absolute))
     {
-        Write-Verbose "Creating NuGet.config file"
-        # Create basic NuGet config file if it doesn't exist
-        [System.Xml.XmlDocument] $nuGetConfig = New-Object System.Xml.XmlDocument
-
-        $configurationSection = $nuGetConfig.CreateElement("configuration")
-        [void]$nuGetConfig.AppendChild($configurationSection)
+        throw (Get-LocalizedString -Key "Feed URI is not properly formatted {0}" -ArgumentList $nugetServer)
     }
-    else
+
+    #check if nuget config exists
+    if((Test-Path -Path $tempNuGetConfigPath))
     {
-        Write-Verbose "Loading existing NuGet.config file"
-        $nuGetConfig = [xml](Get-Content $tempNuGetConfigPath)
-        $configurationSection = $nuGetConfig.configuration
+        Write-Verbose "Deleting previous NuGet.config file"
+        Remove-Item $tempNuGetConfigPath
```

Comment:
Don't bother deleting the old file. Just make sure SetCredentialsNuGetConfigAndSaveTemp will overwrite the existing file.